pageserver/src/tenant/timeline.rs

```diff
@@ -884,7 +891,12 @@ impl Timeline {
     pub async fn shutdown(self: &Arc<Self>, freeze_and_flush: bool) {
         debug_assert_current_span_has_tenant_and_timeline_id();
 
+        // Signal any subscribers to our cancellation token to drop out
+        tracing::info!("Cancelling CancellationToken");
+        self.cancel.cancel();
```

Comment:
This PR changes it how the page service handlers are shutdown. Before they used to be the first ones after stopping the listening task. I think this is good as-is, since this cancellation token is the one used.

However, does the global level still include waiting for these tasks, which are no longer listening to the task_mgr::shutdown_token? Nope, you deleted that, great!

Reply:
Right.  The new flow is:
- Shutdown all tenants (this cancels in-flight page requests)
- Shutdown all page service tasks (this is just joining any leftover tasks that were outside their GateGuard region)